pkg/agent/datapath/policyBridge.go

```diff
@@ -538,6 +562,8 @@ func (p *PolicyBridge) GetTierTable(direction uint8, tier uint8, mode string) (*
 			case POLICY_TIER3:
 				policyTable = p.egressTier3PolicyMonitorTable
 				nextTable = p.egressTier3PolicyTable
+			case POLICY_TIER_ECP:
+				return nil, nil, fmt.Errorf("monitor mode doesn't support tier-ecp")
```

Comment:
We should validate this restriction in webhook, or it will keep retry for 5 mins.

Reply:
done